reactive-messaging-redisstream-connector/src/test/java/hu/icellmobilsoft/reactive/messaging/redis/streams/TestProducer.java

```diff
@@ -25,6 +25,7 @@
 import org.eclipse.microprofile.reactive.messaging.Channel;
 import org.eclipse.microprofile.reactive.messaging.Emitter;
 
+import hu.icellmobilsoft.reactive.messaging.redis.streams.metadata.RedisStreamMetadata;
```

Comment:
Ez tuti kell?

Reply:
Külön package-be lett mozgatva az osztály, így már kell az import.